src/runtime/runtime.js

```diff
@@ -547,7 +547,7 @@ export class ConfiguredRuntime {
     this.offersApi_ = new OffersApi(this.pageConfig_, this.fetcher_);
 
     /** @private @const {!ButtonApi} */
-    this.buttonApi_ = new ButtonApi(this.doc_);
+    this.buttonApi_ = new ButtonApi(this.doc_, opt_integr.configPromise);
```

Comment:
Use Promise.resolve(this) instead.  The one you're passing isn't going to return a runtime object.  Should you add a test for this in runtime-test?

Reply:
Great catch, Matt! Switched to Promise.resolve(this) and added a test.